.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1 +1,26 @@
+<!-- Hey there. Thank you so much for improving NewPipe. Please take a moment to fill out the following suggestion on how to structure this PR description. Having roughly the same layout helps everyone considerably :)-->
+
+#### What is it?
```

Comment:
```suggestion
#### What is it?
<!-- Add an x between the brackets to check them -->
```

Reply:
We aren't doing that at the other templates, see opus's comment above. Why here?

Reply:
Oh yeah, nevermind, sorry 😅

Reply:
Un.be.liev.a.ble.